backends/core/models.py

```diff
@@ -114,14 +138,23 @@ def assign_schedules(self, arg_schedules):
         ids_for_removing.append(schedule.id)
     Schedule.destroy(*ids_for_removing)
 
+  def get_ready(self):
+    cache_values = {
+      "failed_jobs": 0,
```

Comment:
Same as before, please use defined constants for the keys.

Reply:
done